src/framework/control.ml

```diff
@@ -15,28 +15,30 @@ module type S2S = functor (X : Spec) -> Spec
 let spec_module: (module Spec) Lazy.t = lazy (
   GobConfig.building_spec := true;
   let arg_enabled = get_bool "ana.sv-comp.enabled" || get_bool "exp.arg" in
+  let arg_termination = List.mem "termination" (get_string_list "ana.activated") in (* check if loop termination analysis is enabled*)
```

Comment:
Why `arg`? Are you constructing Abstract Reachability Graphs here?

Reply:
we just didn't get the naming convention here :)
Done